### ISSUE TITLE
Untranslated parts of Slicer Tutorials 5.4.0/5.3.0-2023-06-12 - Portuguese

### ISSUE THREAD

**Issue by DouSam** [Slicer/SlicerLanguagePacks#41]
**"Slicer Welcome Tutorial"**

- [ ] Dropdown "Files", option "Download Sample Data"
- [ ] "Help and Acknowledgement" sections from "Sample Data"
- [ ] "BuiltIn", "Development" and the download data progress description 

![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/0db19515-a384-4d9e-aaab-dd1a5d110703)
![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/10a417ac-d898-49a2-9509-d5ebbb1e6185)
![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/9093c795-2e60-4bcd-a852-41263770c596)
![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/07e4f621-12b4-49da-bbf8-a240e216028e)

**"Slicer4 Minute"**

- [ ] "Help and Acknowledgement" sections from "Models"
- [ ] "Diffuse:", "Specular:", "Power:", "Metallic:", "Roughness:", "Preview:" and "Presets" from Model > Display > 3D Display > Advanced
- [ ] Dialog "Select Color"

![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/7b6198f0-fe4d-4c1d-a7ca-74979af48894)
![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/a88fc1eb-6fbe-4839-8f22-2650699cdb82)

**"Data loading and 3D visualization"**

- [ ] DICOM Grid data from "DICOM"
- [ ] Table of colors from "Volume" and the tooltip(when you hover the cursor) from the predefinitions
- [ ] Properties from "Label Properties" from "Volumes"
- [ ] "Help and Acknowledgement" sections from "Volume Rendering"
- [ ] Properties of the Image in "Volume Rendering > Advanced"

![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/860c61df-cae1-4739-a98c-69995565c0da)
![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/bb6f9ee5-37e4-45b1-b574-517654357299)
![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/5aa96246-cf35-4aa3-9c7e-13fe2f9217ec)
![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/85a5cebc-4512-45cb-afd8-b4bb91c9bfc9)
![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/8cc68a94-5ddc-4ff6-bd2d-d88dcda51b61)

**"An Introduction to Image Phenotyping"**

- [ ] "Help and Acknowledgement" sections from "Segment Editor"
- [ ] "Segment" from the output table "Segment Statistics"

![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/2024bed4-819e-4b36-83b6-242ab800e4ee)
![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/5fdccef2-302b-40f3-a510-a66772c2a184)

**Comment by lassoan** [comment]
> "Diffuse:", "Specular:", "Power:", "Metallic:", "Roughness:", "Preview:" and "Presets" from Model > Display > 3D Display > Advanced

Need to translate in CTK:
https://hosted.weblate.org/translate/3d-slicer/ctk/hu/?q=diffuse&sort_by=-priority%2Cposition&checksum=

> Dialog "Select Color"

It is probably provided by the operating system. Please create a separate issue to investigate translation of native dialogs (File open/save, select color, etc.) and here add the link to the new issue and mark it "completed".

>  DICOM Grid data from "DICOM"

Need to translate in CTK.

> Table of colors from "Volume" and the tooltip(when you hover the cursor) from the predefinitions

Need to implement translation for `Modules\Loadable\Volumes\Resources\VolumeDisplayPresets.json` - similarly to how CLI module descriptor .xml files are translated. Please create a separate issue for this and here add the link to the new issue and mark it "completed".

> Properties from "Label Properties" from "Volumes"

Need to translate in CTK.

>  Properties of the Image in "Volume Rendering > Advanced"

Need to translate in CTK.

> "Segment" from the output table "Segment Statistics"

This is tracked here: https://github.com/Slicer/Slicer/issues/7217
We can mark the issue completed here and add this link.

Everything else I'll address in a commit today.

**Comment by DouSam** [comment]
I have opened issues #42 and #43, for the specific situations. Thanks for reminding me from the CTK, I was searching only in "3D Slicer" Weblate component.

**Comment by lassoan** [comment]
My pull request that marks the missing strings as translatable has been merged and the new strings now show up on weblate so they can be translated. To see the strings translated in the application you need to use a Slicer Preview Release that is downloaded tomorrow or later.

After you confirm that a string appears as translated in the application, check the corresponding checkbox in the list above. If everything goes well then all the checkboxes will be checked and the issue can be closed.

**Comment by DouSam** [comment]
@lassoan Could you please check the Acknowledgment section from "Models"? Apparently, there is no key for this section
![image](https://github.com/Slicer/SlicerLanguagePacks/assets/28208639/23a863b0-25d0-4746-916c-b213270c2d9b)

**Comment by lassoan** [comment]
Good catch, Models' module acknowledgment text was not marked as translatable. I'll push a fix today.
